Extensions/mutualchecker.js

```diff
@@ -67,7 +71,7 @@ XKit.extensions.mutualchecker = new Object({
 				}
 				XKit.extensions.mutualchecker.check(json_obj, $post.find(".post_info_submissions").first());
 			} else {
-				$post.find(".post_info_link[data-tumblelog-popover]:not(.reblog_icon + .post_info_link), .post-info-tumblelog > a[data-tumblelog-popover]").each(function() {
+				$post.find(".post_info_link[data-tumblelog-popover]:not(.reblog_icon + .post_info_link):not(.tf2_icon + .post_info_link), .post-info-tumblelog > a[data-tumblelog-popover]").each(function() {
```

Comment:
what's going on with this change?

Reply:
prevents mutualchecker from checking "reblogged from" URLs when TF2 reblogs has already completed on the post